tests/testData/annotation/declaration_missing_assignment.wdl

```diff
@@ -0,0 +1,4 @@
+version draft-3
+workflow foo {
+  <error descr="Immediate assignment required for non-input declaration [draft-3]">Int i</error>
+}
```

Comment:
Prefer newlines at the ends of files if possible (@kshakir has particularly strong opinions on this... 😄)